### PR TITLE
[jenkins] updating installation process

### PR DESCRIPTION
**Pull request by l0k0ms**
https://trello.com/c/kojYK7QI/

### CODE REVIEW

File: content/integrations/jenkins.md [hunk 1]
```diff
@@ -9,29 +9,31 @@ newhlevel: true
 ## Overview
 
 {{< img src="integrations/jenkins/integration-jenkins-overview.png" alt="Jenkins Plugin" >}}
-A Jenkins plugin used to forward metrics, events, and service checks to an account at Datadog, automatically.
+A Jenkins plugin used to forward metrics, events, and service checks to an account at Datadog, automatically.
```
Comment (kmshultz) [inline content/integrations/jenkins.md]: I think we can tighten up the whole Overview. In particular the **Note** is too wordy.

---
This [Jenkins plugin](https://github.com/DataDog/jenkins-datadog-plugin) sends your build and deployment events to Datadog. From there, you can overlay them onto graphs of your other metrics so you can identify which deployments really affect your application's performance and reliability—for better or worse.

**Note: The Jenkins check for the Datadog Agent is deprecated.** Use the Jenkins plugin.

**Comment by kmshultz** [comment]
Also according to https://github.com/DataDog/jenkins-datadog-plugin the plugin does submit service checks too (let's add a section here)

File: content/integrations/jenkins.md (hunk 1 above)
Comment (l0k0ms) [inline content/integrations/jenkins.md]: :+1

**Comment by l0k0ms** [comment]
@kmshultz PR updated.

File: content/integrations/jenkins.md [hunk 2]
```diff
@@ -6,32 +6,36 @@ git_integration_title: jenkins
...
 ## Overview
 
-{{< img src="integrations/jenkins/integration-jenkins-overview.png" alt="Jenkins Plugin" >}}
-A Jenkins plugin used to forward metrics, events, and service checks to an account at Datadog, automatically.
+This Jenkins plugin sends your build and deployment events to Datadog. From there, you can overlay them onto graphs of your other metrics so you can identify which deployments really affect your application's performance and reliability—for better or worse.
 
-***Note: The Jenkins Agent Integration is Deprecated***: The integration in the Datadog agent for Jenkins has been deprecated in favor of the open-source Jenkins plugin. This document refers to installing the Jenkins plugin. For more details about the plugin, refer to the [GitHub repo](https://github.com/DataDog/jenkins-datadog-plugin).
+The plugin also tracks build times (as a metric) and statuses (as a service check), so you'll know when your builds aren't healthy.
+
+**Note: The Jenkins check for the Datadog Agent is deprecated.** Use the [Jenkins plugin](https://github.com/DataDog/jenkins-datadog-plugin).
```
Comment (kmshultz) [inline content/integrations/jenkins.md]: Come to think of it, I don't think we should link to the code here. It tempts the reader to jump to the Github doc right before starting to read _this_ doc.

We _should_ link to it somewhere. Further Reading?

Comment (l0k0ms) [inline content/integrations/jenkins.md]: I'm not so sure, we need to warn in the overview that our jenkins check is deprecated. And by doing so we have to talk about the jenkins plugin, but omitting the link here would be weird I think.

I would keep it that way and still add a link to the plugin in the further reading section. What do you think ?
